### PR TITLE
Explicitly set the trace sampling to 10%

### PR DESCRIPTION
**Pull request by tcnghia**

### CODE REVIEW

File: pkg/httpmetrics/metrics.go [hunk 1]
```diff
@@ -255,6 +255,8 @@ func tracerOptionsGCP(ctx context.Context) []trace.TracerProviderOption {
 	return []trace.TracerProviderOption{
 		trace.WithResource(res),
 		trace.WithSpanProcessor(bsp),
+		// Use 0.1 percent.
```
Comment (cmdpdx) [inline pkg/httpmetrics/metrics.go:258]:
```suggestion
		// Use 10 percent.
```
nit

File: pkg/httpmetrics/metrics.go [hunk 2]
```diff
@@ -255,6 +255,8 @@ func tracerOptionsGCP(ctx context.Context) []trace.TracerProviderOption {
...
+		trace.WithSampler(trace.ParentBased(trace.TraceIDRatioBased(0.1))),
```
Comment (k4leung4) [inline pkg/httpmetrics/metrics.go:259]: can we make this a var or env?

Comment (tcnghia) [reply pkg/httpmetrics/metrics.go]: Done

**Review by k4leung4 — APPROVED** [review]
thank you

**Pull request merged**